opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -1800,6 +1800,47 @@ namespace detail {
             }
         }
 
+        /// \brief Compute the L-infinity norm of a vector
+        /// \warn This function is not suitable to compute on the well equations.
+        /// \param a The container to compute the infinity norm on.
```

Comment:
Also, it should be explained that the components are assumed to be consecutive (all data for the first component before the data for the second component etc.).